### DISCUSSION TITLE
Spotify Widget

### DISCUSSION THREAD

**Discussion by Leoner35** [ShankarBUS/ModernFlyouts#783] [Ideas]
That it can be configured so that if you are listening to music the spotify widget does not remove itself after a few seconds, and do it as if it were a desktop widget.

**Comment by prakharb5** [comment]
We already have a feature request #368 for that.

**Comment by victorjon68** [comment]
what about a spotify button on the "flyout" to add the playing song to to liked songs or remove it if it is already liked? that would be great, otherwise this right here is great work you guys have done.
